### PR TITLE
series drop nulls documentation

### PR DESCRIPTION
**Pull request by hathawayj**
I moved over to the series methods and tried the documentation for `drop_nulls()`. Followed directions in https://github.com/narwhals-dev/narwhals/issues/71.

### CODE REVIEW

**Review by MarcoGorelli — COMMENTED** [review]
That was fast - nice one!

This is close - however, `nw.col(column).drop_nulls()` is actually `Expr.drop_nulls`, not `Series.drop_nulls`. `nw.col(column)` is an `Expr`

It should be a simple change though:
- instead of making `df_pd` and `df_pl`, you can make `s_pd` and `s_pl`
- when you call `nw.from_native`, pass `series_only=True`

So, your function will look something like
```python
def func(s_any):
    s = nw.from_native(s_any, series_only=True)
    s = s.drop_nulls()
    return nw.to_native(s)
```

**Comment by hathawayj** [comment]
Yeah. I was wondering about that.  I will update.

File: narwhals/series.py [hunk 1]
```diff
@@ -123,6 +123,49 @@ def is_in(self, other: Any) -> Self:
...
+
+          Then we can pass either Series (polars or pandas) to `func`:
+
+          >>> func(s_pd)
+          0    2.0
+          1    4.0
+          3    3.0
+          4    5.0
+          dtype: float64
+          >>> func(s_pl)
```
Comment (MarcoGorelli) [inline narwhals/series.py:159]:
```suggestion
          >>> func(s_pl)  # doctest:+SKIP
```

**Review by MarcoGorelli — APPROVED** [review]
thanks @hathawayj for your PR!

I came across this today too, and can't tell what the cause it. The docstring validation works fine for `dataframe.py` and `expression.py`, but not for `series.py`

For now, let's just add a `  # doctest: +SKIP` for the Polars one, but hopefully we can figure out why this is happening and remove it later

**Comment by MarcoGorelli** [comment]
> For now, let's just add a   # doctest: +SKIP for the Polars one, but hopefully we can figure out why this is happening and remove it later

Found it - the Polars output uses tabs instead of spaces (!)

So I've set `# doctest: +NORMALIZE_WHITESPACE` for the Polars Series outputs

**Pull request merged**